### ISSUE TITLE
Weird issue with markdown preview tab

### ISSUE THREAD

**Issue by BolajiAyodeji** [acekyd/made-in-nigeria#317]
### Issue Description

The preview tab of the `contributing.md` file weirdly doesn't exist unlike `README.MD` making the "Make a Submission" link from the website lead to the raw markdown without any rendering as seen in the screenshot below. I tried reproducing on a new repository but it's not the same—all three tabs are displayed (Preview, Code, and Blame). Also, it's not because it's `.md` and not `.MD` (both ought to work).

My guess is that the files have not been updated for a long time (maybe update it?) or there's a setting you need to toggle in GitHub. But this is worth investigating to offer a better reading experience.

<img width="1122" alt="Screenshot 2024-03-13 at 17 21 25" src="https://github.com/acekyd/made-in-nigeria/assets/30334776/701f2e77-5a3b-487c-a810-45dd2113bd70">

<img width="580" alt="Screenshot 2024-03-13 at 17 38 54" src="https://github.com/acekyd/made-in-nigeria/assets/30334776/16e32521-d2c7-440a-bf1e-920461963f22"

### Expected Behavior

`contributing.md` and `moderators.md` files should include the "Preview" tab by default.

### Please check the boxes that apply to this issue report.

- [ ] I have searched the existing issues to make sure this is not a duplicate.
- [ ] I have provided steps to reproduce the issue.
- [ ] I have included relevant environment information.
- [ ] I have included any relevant screenshots.
- [ ] I understand that this is a voluntary contribution and that there is no guarantee of resolution.
- [ ] I want to work on creating a PR for this issue if approved

**Comment by kaf-lamed-beyt** [comment]
This could be as a result of the way the filename is spelled. I'll try making both of them uppercase and see if the issue still persists.

**Comment by kaf-lamed-beyt** [comment]
> This could be as a result of the way the filename is spelled. I'll try making both of them uppercase and see if the issue still persists.

Case sensitivity does not affect the rendering engine. It could be a syntax issue. I'm taking a careful look at both files again.

**Comment by BolajiAyodeji** [comment]
Yes, markdown files are not case-sensitive on GitHub for both the name and file extension. I did try to reproduce to be sure of this and both worked in another repository. Something else seems to be at play here. Curious to see what you find out, thank you.
